Lib/ufo2ft/util.py

```diff
@@ -32,15 +32,20 @@ def makeOfficialGlyphOrder(font, glyphOrder=None):
     If not explicit glyphOrder is defined, sort glyphs alphabetically.
 
     If ".notdef" glyph is present in the font, force this to always be
-    the first glyph (at index 0).
+    the first glyph (at index 0). Also, if "space" is present in the font and
+    missing from glyphOrder, force it to be the second glyph (at index 1).
```

Comment:
I'm now thinking that this doesn't check that the glyph named 'space' is actually empty before putting it at index 1, and the whole point of doing this (which isn't even mentioned here, I suppose it should) was to make sure COLRv0 fonts work in old Windows. Maybe it should just move the first empty glyph it finds to second place, whatever its name.